requirements-tests.txt

```diff
@@ -1,21 +1,30 @@
-aiohttp==3.8.4; python_version < "3.12"           # aiohttp can't be installed on 3.12 yet
+# Type checkers and other linters that we test our stubs against. These should always
+# be pinned to a specific version to make failure reproducible. See also the
+# "tool.typeshed" section in pyproject.toml for additional type checkers.
 black==23.3.0                                     # must match .pre-commit-config.yaml
 flake8==6.0.0; python_version >= "3.8"            # must match .pre-commit-config.yaml
 flake8-bugbear==23.5.9; python_version >= "3.8"   # must match .pre-commit-config.yaml
 flake8-noqa==1.3.1; python_version >= "3.8"       # must match .pre-commit-config.yaml
 flake8-pyi==23.5.0; python_version >= "3.8"       # must match .pre-commit-config.yaml
 isort==5.12.0; python_version >= "3.8"            # must match .pre-commit-config.yaml
 mypy==1.3.0
-packaging==23.1
-pathspec>=0.11.1
-pre-commit-hooks==4.4.0                           # must match .pre-commit-config.yaml
 pycln==2.1.5                                      # must match .pre-commit-config.yaml
 pytype==2023.6.2; platform_system != "Windows" and python_version < "3.11"
+
+# Git support.
+pre-commit-hooks==4.4.0                           # must match .pre-commit-config.yaml
```

Comment:
I'd put this in the "linters" category -- it's only there so that people can run these hooks manually in their local environments, if they want to:

https://github.com/python/typeshed/blob/9ca472420429c36317f280cad9db81eb72e2f5ac/.pre-commit-config.yaml#L5-L12